client/login/magic-login/request-login-email-form.jsx

```diff
@@ -124,6 +174,12 @@ class RequestLoginEmailForm extends Component {
 
 		const usernameOrEmail = this.getUsernameOrEmailFromState();
 
+		const siteIcon =
+			this.state.site?.icon?.ico ??
+			this.state.site?.icon?.img ??
+			this.state.site?.logo?.url ??
+			null;
```

Comment:
Will all these always be squares, so that we don't accidentally stretch the icon into 64x64?

Reply:
I removed logo as a fallback. The site icon itself could be square, but can also be other shapes depending on how a user sets it. For now, I've tweaked code to not assume an icon is always square. Slack discussion: p1699478785580739-slack-C03N25JPCE4